Xamarin.Forms.Core/Storyboard.cs

```diff
@@ -0,0 +1,551 @@
+		}
+
+		public static void SetTargetProperty(BindableObject bindable, BindableProperty value)
+		{
+			bindable.SetValue(TargetPropertyProperty, value);
+		}
+
+		protected override void OnBindingContextChanged()
+		{
+			base.OnBindingContextChanged();
+			foreach (var item in Children)
+			{
+				item.SetValue(BindingContextProperty, BindingContext);
+				//SetInheritedBindingContext(item, BindingContext);
+
+			}
+		}
+
+		public static BindableProperty GetTargetProperty(BindableObject bindable)
+		{
+			return (BindableProperty)bindable.GetValue(TargetPropertyProperty);
+		}
+
+		public Storyboard()
+		{
+			_timelines = new List<Timeline>();
+		}
+
+		public void Begin(Timeline item)
+		{
+			_animationsPerElement.Clear();
+			_runningAnimations.Clear();
+			var maxDuration = Duration;
+			var defaultTarget = Storyboard.GetTarget(this);
+			var defaultTargetProperty = Storyboard.GetTargetProperty(this);
+
+
+			if (item is Animation<Color> timeline)
+			{
+				maxDuration = AddAnimationColor(maxDuration, defaultTarget, defaultTargetProperty, timeline);
+			}
+
+			if (item is DoubleAnimation doubleAnimation)
+			{
+				AddDoubleAnimation(defaultTarget, defaultTargetProperty, doubleAnimation);
+			}
+
+			if (item is PositionAnimation positionAnimation)
+			{
+				AddPositionAnimation(defaultTarget, defaultTargetProperty, positionAnimation);
+			}
+			foreach (var anim in _animationsPerElement)
+			{
+				var elementAnimation = new Animation();
+				foreach (var animation in anim.Value)
+				{
+					elementAnimation.Add(0, 1, animation);
+
+				}
+				_runningAnimations.Add(elementAnimation);
+				elementAnimation.Commit(anim.Key, "ChildAnimations", 16, maxDuration, finished: (v, c) => Completed?.Invoke(this, new EventArgs()));
+			}
+		}
+
+		public new void Begin()
+		{
+			_animationsPerElement.Clear();
+			_runningAnimations.Clear();
+			var maxDuration = Duration;
+			var defaultTarget = Storyboard.GetTarget(this);
+			var defaultTargetProperty = Storyboard.GetTargetProperty(this);
+			foreach (var item in Children)
+			{
+				if (item is Animation<Color> timeline)
+				{
+					maxDuration = AddAnimationColor(maxDuration, defaultTarget, defaultTargetProperty, timeline);
+				}
+
+				if (item is DoubleAnimation doubleAnimation)
+				{
+					AddDoubleAnimation(defaultTarget, defaultTargetProperty, doubleAnimation);
+				}
+
+				if (item is PositionAnimation positionAnimation)
+				{
+					AddPositionAnimation(defaultTarget, defaultTargetProperty, positionAnimation);
+				}
+
+			}
+
+			foreach (var anim in _animationsPerElement)
+			{
+				var elementAnimation = new Animation();
+				foreach (var animation in anim.Value)
+				{
+					if (Reverse)
+					{
+						elementAnimation.Add(0, 0.5, animation);
+						elementAnimation.Add(0.5, 1, new Animation(animation.GetCallback(), 1, 0, Easing));
+					}
+					else
+					{
+						elementAnimation.Add(0, 1, animation);
+					}
+						
```

Comment:
When I format Storyboard.cs a difference appears just because of this line... I wish all code was formatted before committing otherwise any future commit will create a difference. And I had to work hard on fixing these differences and not to trigger changes because of formatting the code

Reply:
I might be having something wrong on my preview machine. You are right ,  we need to add something to the repo to verify style, I think I saw something from the NETCore team that does this.

Reply:
This would be truly glorious to get. It might seem like a little thing, but it's so important on the long run, it saves a great amount of time for everybody.